README.md

```diff
@@ -1,33 +1,147 @@
-# Blank Next.js Space Blueprint
 
-This is a [Next.js](https://nextjs.org) project integrated with Storyblok. It is designed to work with a blank Storyblok space, supporting the default `page`, `teaser`, `grid`, and `feature` components. The Visual Editor's live preview works out of the box. The project comes with minimal styling and is ideal to kickstart your new Storyblok and Next.js project.
+# Astro Storyblok Starter Blueprint
 
-## Getting Started
+Integrate [Astro](https://astro.build/) and [Storyblok](http://www.storyblok.com) as a headless CMS.
 
-First, install all dependencies:
-```bash
+The blueprint is ideal for kick-starting new Storyblok and Astro projects. It features the following:
+- Pre-configured default blocks: `page`, `teaser`, `grid`, and `feature`.
+- Support for the Visual Editor's live preview.
+- Dynamic routing to fetch and render new stories automatically.
+- Rich text rendering and global content management.
+- Authentication-ready.
+- Internationalization-ready.
+- Minimal styling.
+
+> [!SUMMARY] Live preview
```

Comment:
Do we need to mention it specifically? If we link to the related tech guide, live preview will be explained anyway.